benchopt/utils/dependencies_mixin.py

```diff
@@ -159,7 +161,11 @@ def collect(cls, env_name=None, force=False):
             post_install_hooks = [cls._post_install_hook]
         else:
             env_suffix = f" in '{env_name}'" if env_name else ''
-            print(f"- '{cls.name}' already available{env_suffix}")
+            colored_cls_name = colorify(f'{cls.name}', YELLOW)
+            print(
+                f"- {colored_cls_name} already available{env_suffix}\n"
+                "  All imports within `safe_import_context` are installed."
```

Comment:
RE-reading the comment from @mathurinm, maybe this is clearer, less `benchopt` jargon:
```suggestion
                f"  No ImportError raised from {cls.module_filename}."
```

Reply:
The CI fails.
From the error logs, it appears that not all objects have ``module_filename``
```bash
=========================== short test summary info ============================
FAILED benchopt/tests/test_cli.py::TestInstallCmd::test_benchopt_install - AttributeError: type object 'Objective' has no attribute 'module_filename'
FAILED benchopt/tests/test_cli.py::TestInstallCmd::test_benchopt_install_in_env - AttributeError: type object 'Objective' has no attribute 'module_filename'
FAILED benchopt/tests/test_cli.py::TestInstallCmd::test_benchopt_install_in_env_with_requirements - AttributeError: type object 'Solver' has no attribute 'module_filename'
======= 3 failed, 139 passed, 6 skipped, 7 warnings in 194.93s (0:03:14) =======
Error: Process completed with exit code 1.
```